lang/LangPrimSource/PyrStringPrim.cpp

```diff
@@ -622,10 +622,13 @@ int prString_Getenv(struct VMGlobals* g, int /* numArgsPushed */) {
         return this_err;
 
 #ifdef _WIN32
-    const auto count = GetEnvironmentVariable(this_str.c_str(), nullptr, 0);
-    std::string buf(count, 0);
+    const auto this_str_w = SC_Codecvt::utf8_cstr_to_utf16_wstring(this_str.c_str());
+    const auto count = GetEnvironmentVariableW(this_str_w.c_str(), nullptr, 0);
+    std::string buf;
     if (count != 0) {
-        GetEnvironmentVariable(this_str.c_str(), buf.data(), buf.size());
+        std::wstring wbuf(count, 0);
+        GetEnvironmentVariableW(this_str_w.c_str(), &wbuf[0], count);
+        buf = SC_Codecvt::utf16_wcstr_to_utf8_string(wbuf.c_str());
     }
     char* value = count != 0 ? buf.data() : nullptr;
```

Comment:
@JordanHendersonMusic I was wondering, now that I'm getting `buf` string from the `utf16_wcstr_to_utf8_string` function, should I use `buf.c_str()` here instead of `.data()`?

Reply:
Upon further reflection, I think this should indeed be `buf.c_str()` so I'm changing this.

Reply:
Well... it seems that doesn't work, after all. I got 
`D:\a\supercollider\supercollider\lang\LangPrimSource\PyrStringPrim.cpp(633,53): error C2440: 'initializing': cannot convert from 'const _Elem *' to 'char *' `
when compiling.
Reverting back to `.data()`.